static/js/index.js

```diff
@@ -2,9 +2,10 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import { HashRouter, Route } from "react-router-dom";
+import color from 'color';
 
 import '../css/index.scss';
-import Sidebar from './components/sidebar.js';
+import Sidebar from './components/Sidebar/sidebar.js';
```

Comment:
I believe this should be imported from `./components/Sidebar/Sidebar.js`

Reply:
Gah, it was until something messed that up... Thanks for catching that!

Reply:
OK, fixed!